From 246f824a6a0d2bc93ec4ad2ebfde7c92b42c879a Mon Sep 17 00:00:00 2001
From: Alfred Wingate <parona@protonmail.com>
Date: Tue, 21 Nov 2023 01:10:23 +0200
Subject: [PATCH] Fix detection of QtDBus and QtMultimedia

* Sometimes you may have qtdbus and qtmultimedia present at the same
  time.

Signed-off-by: Alfred Wingate <parona@protonmail.com>
--- a/ac/qt6.m4
+++ b/ac/qt6.m4
@@ -95,10 +95,12 @@ EOT
 
     if test $result2 != 0; then
       continue
-    elif test $qt_module = dbus; then
+    fi
+    if test $qt_module = dbus; then
       qmake_qt_ui="$qmake_qt_ui dbus"
       AC_DEFINE(HAVE_QTDBUS, 1, [Define if QtDBus is present])
-    elif test $qt_module = multimedia; then
+    fi
+    if test $qt_module = multimedia; then
       qmake_qt_ui="$qmake_qt_ui multimedia"
     fi
   done
-- 
2.43.0

